cat/ptx-proxy.cat

```diff
@@ -78,13 +80,14 @@ let cause-write = ([W]; cause ; [W]) & loc
 empty (cause-write \ co) as axiom-Coherence
 
 (* Axiom 2 FenceSC *)
-//this is equivalent to irreflexive (fence-sc ; cause) as axiom-FenceSC
+//this is equivalent to: irreflexive (fence-sc ; cause) as axiom-FenceSC
 empty ([F & SC] ; cause; [F & SC] \ fence-sc) as axiom-FenceSC
 
 (* Axiom 3 Atomicity *)
 empty (((morally-strong & fr); (morally-strong & co)) & rmw) as axiom-Atomicity
 
 (* Axiom 4 No-Thin-Air *)
+// The alloy model does not support the dep used below
```

Comment:
This is not a problem of their model, but rather of their tool (which does not properly populate the base relations). I don't think we should document problems with their **tool**. The proper way of doing this is by opening an issue in their repository (as you already did).

Reply:
Removed